### DISCUSSION TITLE
How to register my own provider's MemcpyToHost/MemcpyFromHost Op

### DISCUSSION THREAD

**Discussion by lewisword** [microsoft/onnxruntime#5535] [Other Q&A]
**problem describe:**
I follow the instruction of [AddingExecutionProvider](https://github.com/microsoft/onnxruntime/blob/master/docs/AddingExecutionProvider.md), and successfully register my own provider and some simple op(Add .etc). But when I run the onnx model which contains op both from CPU and my own provider, onnxruntime sends me one error that my own provider doesn't have MemcpyToHost/MemcpyFromHost Op.So how to?
![批注](https://user-images.githubusercontent.com/31215614/96423161-82115600-122b-11eb-89ac-78d7177c862d.png)

**System information:**
onnxruntimev1.5.1

**Additional context:**
* I try to find the definition of MemcpyToHost/MemcpyFromHost in cuda, but fail.
* I try to register MemcpyToHost/MemcpyFromHost just like other common op(for example, Add), but fail

**Comment by galdav117** [comment]
I am encountering the exact same problem, could you share how you solved it?

Thanks!
